Okta.AspNetCore/OpenIdConnectOptionsHelper.cs

```diff
@@ -37,7 +37,9 @@ public static void ConfigureOpenIdConnectOptions(OktaMvcOptions oktaMvcOptions,
             oidcOptions.SignedOutRedirectUri = oktaMvcOptions.PostLogoutRedirectUri;
             oidcOptions.ResponseType = OpenIdConnectResponseType.Code;
             oidcOptions.GetClaimsFromUserInfoEndpoint = oktaMvcOptions.GetClaimsFromUserInfoEndpoint;
+
 #if NET8_0_OR_GREATER
+            oidcOptions.UseSecurityTokenValidator = true;
             oidcOptions.TokenHandler = new StrictTokenHandler();
```

Comment:
@laura-rodriguez , thank you for looking into this.

In my work-around in #260 : 
I continue to use the Okta-supplied validation path, by explicitly calling it -- much like the dot-net code would have if `UseSecurityTokenValidator==false`.

A second validation is then done by dotnet because I set `UseSecurityTokenValidator=true`.

I believe that setting `oidcOptions.UseSecurityTokenValidator = true;` here, the dotnet code will never call the Okta-supplied `StrictTokenHandler` because it will never call `ValidateTokenUsingHandlerAsync` within OpenIdConnectHandler.cs#L532.

Can you confirm that it is desired by Okta, when running in net8.0, that only the default dotnet validator is used.?

If so, then the line `oidcOptions.TokenHandler = new StrictTokenHandler();` is not needed here as it is never called.
Otherwise, another approach will be needed.

Reply:
Hi @NickDrouin,

Excellent point. You're right; I'll be pushing an update soon.

Thank you

Reply:
Agree with @NickDrouin that setting UseSecurityTokenValidator to true will never call any of the Okta custom handlers as nothing was ever set for the SecurityTokenValidator property.

IMHO, To make this library NET8.0 compliant, a new class should be added to handle token validation for the JsonWebToken type.  This new class would be derived from the JsonWebTokenHandler class and then assigned to the TokenHandler property on the oidcOptions.  This new class would override the ValidateToken and ValidateTokenAsync methods to add additional validation for the RS256 algorithm requirement, similar to the existing StrictTokenHandler class.

Reply:
Hey @GeoRHeld,

Thank you for your feedback. Due to timeline concerns and to unblock our customers as soon as possible, we'll patch this release using the `UseSecurityTokenValidator` setting. I'll file a follow-up internal ticket to be fully .NET 8 compliant and add all the necessary testing to ensure everything's working as expected.